pywinauto/unittests/test_win32controls.py

```diff
@@ -54,110 +54,87 @@
 
 
 class ButtonTestCases(unittest.TestCase):
-
     """Unit tests for the ButtonWrapper class"""
 
     def setUp(self):
         """Start the application set some data and ensure the application
         is in the state we want it."""
 
         self.app = Application()
+        self.app = self.app.Start(os.path.join(mfc_samples_folder, u"CmnCtrl1.exe"))
 
-        if is_x64_Python() or not is_x64_OS():
-            self.app.start(r"C:\Windows\System32\calc.exe")
-        else:
-            self.app.start(r"C:\Windows\SysWOW64\calc.exe")
-        self.calc = self.app.Calculator
-        self.calc.MenuSelect("View->Scientific")
+        self.app.Common_Controls_Sample.TabControl.Select("CDateTimeCtrl")
 
-    def tearDown(self):
-        "Close the application after tests"
+        self.ctrl = self.app.Common_Controls_Sample
 
+    def tearDown(self):
+        """Close the application after tests"""
         self.app.kill_()
-        #self.calc.type_keys("%{F4}")
 
     def testGetProperties(self):
-        "Test getting the properties for the button control"
-        props = self.calc.Degrees.GetProperties()
+        """Test getting the properties for the button control"""
+        props = self.ctrl.Button2.GetProperties()
 
         self.assertEquals(
-            "RadioButton", props['friendly_class_name'])
+            "Button", props['friendly_class_name'])
 
         self.assertEquals(
-            self.calc.Degrees.texts(), ['Degrees'])
+            self.ctrl.Button2.texts(), ['Отмена'])

```

Comment:
The tests must work on Windows with English or any other interface language. Please choose another control with English-only names.